cmd/boulder-publisher/main.go

```diff
@@ -83,6 +84,13 @@ func main() {
 	gw := bgrpc.NewPublisherServerWrapper(pubi)
 	pubPB.RegisterPublisherServer(grpcSrv, gw)
 
+	go func() {
```

Comment:
Can you add a comment here? I think its clear what the code is doing but maybe doesn't capture why. I think a mention of our intentions w.r.t capturing some debug information from the context of the publisher would be helpful if someone stumbles across this down the road.